### DISCUSSION TITLE
random insignificant questions

### DISCUSSION THREAD

**Discussion by stysedo** [samschott/maestral#542] [Q&A] [answered]
A couple of random questions about Maestral on macOS...

-If we are only planning to ever use Maestral.app via the GUI is it necessary to install the CLI tools?

-Is the directory at  ~/.dropbox part of Maestral or is it likely a relic of having had the official Dropbox.app running in the past?

**Comment by samschott** [comment, marked as answer]
> If we are only planning to ever use Maestral.app via the GUI is it necessary to install the CLI tools?

No, not necessary. Though the CLI is always bundled with the app. Installing it only makes it more easily accessible by creating a link at `/usr/local/bin` where most command line programs live.

> Is the directory at ~/.dropbox part of Maestral or is it likely a relic of having had the official Dropbox.app running in the past?

It's from the official client. Maestral settings on macOS are stored at `~/Library/Application Support/maestral`.

**Reply by stysedo** [reply]
I think I'll do a bit of tidying up now!
